package.json

```diff
@@ -1,34 +1,39 @@
 {
```

Comment:
My atom must have automatically reformatted the spacing.  I can undo this and only add my deltas if required.